cmd/sos_sync.go

```diff
@@ -0,0 +1,518 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"github.com/minio/minio-go/v6"
+	"github.com/vbauerster/mpb/v4"
+	"github.com/vbauerster/mpb/v4/decor"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+//region Settings
+
+const (
+	defaultParallelSosSync = 10
+)
+
+//endregion
+
+//region Type definitions
+
+type sosSyncClientFactory = func(certsFile string) (*sosClient, error)
```

Comment:
It looks like you have somewhat of a Java background ;)
We strive to implement our Go project with the most idiomatic styles, namely:

* https://golang.org/doc/effective_go.html
* https://github.com/uber-go/guide/blob/master/style.md

For consistency's sake, could you please try to adopt a similar style to the rest of the codebase, so there is no additional cognitive load when operating across the entire project?